airbyte-cdk/python/setup.py

```diff
@@ -66,7 +66,7 @@
     packages=find_packages(exclude=("unit_tests",)),
     install_requires=[
         "backoff",
-        "jsonschema==2.6.0",
+        "jsonschema==3.2.0",
```

Comment:
do we need `==` or can we use `~=` ? It's always better to use that to allow consumers of this package flexibility in package versions they leverage

Reply:
Ok